### DISCUSSION TITLE
Installazione piattaforma

### DISCUSSION THREAD

**Discussion by obiettivos** [globaleaks/globaleaks-whistleblowing-software#3891] [Q&A]
Ho chiesto a Serverplan (provider per il sito per cui vorrei attivare Globaleaks) indicazioni su come procedere. Di seguito la loro risposta:

Buongiorno,
Sui nostri server l’installazione del software GlobaLeaks non è consentita.
Restiamo a disposizione,
Saluti,

Come posso procedere?

Grazie, Riccardo.

**Comment by evilaliv3** [comment]
Buonasera.

Se vuole può scriverci ad info@globaleaks.org con le sue necessità e possiamo farle seguito con una offerta da parte della nostra impresa sociale non profit. Ogni ricavato viene reinvestito nella crescita del progetto.

**Reply by obiettivos** [reply]
Ma l'offerta che vorreste farmi è relativa ad un vostro servizio di hosting per la piattaforma?

**Reply by evilaliv3** [reply]
Se siete interessati al solo hosting, un qualsivoglia fornitore di server dedicati o server virtuali  debian/linux va bene dal punto di vista dei requisiti tecnici.

Laddove necessario la nostra impresa sociale eroga servizi di consulenza professionale e soluzioni SaaS.

**Comment by spomar** [comment]
un qualsiasi altro service provider che non abbia questo limite, esistono ma non e' elegante fare qui pubblica promozione.
